client/src/components/Collections/ListCollectionCreator.vue

```diff
@@ -10,7 +10,7 @@
                 <b-alert show variant="warning" dismissible>
                     {{ noElementsHeader }}
                     {{ allInvalidElementsPartOne }}
-                    <a class="cancel-text" href="javascript:void(0)" role="button" @click="oncancel">
+                    <a class="cancel-text" href="" role="button" @click.prevent="oncancel">
```

Comment:
Using a button for links without an href is preferable. The `.ui-link` helper class should style a button to look like a link.